### PR TITLE
Update prices_arbitrum_tokens.sql

### PR DESCRIPTION
**Pull request by kvtoraman**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.
Add native USDC by Circle (0xaf88d065e77c8cc2239327c5edb3a432268e5831)

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by kvtoraman** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by kvtoraman** [comment]
recheck

**Comment by kvtoraman** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by kvtoraman** [comment]
recheck

**Comment by kvtoraman** [comment]
@couralex6 Hi Alex, can you check my PR?

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -83,7 +83,8 @@ FROM
     ('uma-uma','arbitrum','UMA',0xd693ec944a85eeca4247ec1c3b130dca9b0c3b22,18),
     ('umami-umami-finance','arbitrum','UMAMI',0x1622bf67e6e5747b81866fe0b85178a93c7f86e3,9),
     ('uni-uniswap','arbitrum','UNI',0xfa7f8980b0f1e64a2062791cc3b0871572f1f7f0,18),
-    ('usdc-usd-coin', 'arbitrum', 'USDC', 0xff970a61a04b1ca14834a43f5de4533ebddb5cc8, 6),
+    ('usdc-usd-coin', 'arbitrum', 'USDC.e', 0xff970a61a04b1ca14834a43f5de4533ebddb5cc8, 6),
+    ('usdc-usd-coin', 'arbitrum', 'USDC', 0xaf88d065e77c8cC2239327C5EDb3A432268e5831, 6),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:87]: arbiscan: https://arbiscan.io/token/0xaf88d065e77c8cC2239327C5EDb3A432268e5831

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 2]
```diff
@@ -83,7 +83,8 @@ FROM
     ('uma-uma','arbitrum','UMA',0xd693ec944a85eeca4247ec1c3b130dca9b0c3b22,18),
     ('umami-umami-finance','arbitrum','UMAMI',0x1622bf67e6e5747b81866fe0b85178a93c7f86e3,9),
     ('uni-uniswap','arbitrum','UNI',0xfa7f8980b0f1e64a2062791cc3b0871572f1f7f0,18),
-    ('usdc-usd-coin', 'arbitrum', 'USDC', 0xff970a61a04b1ca14834a43f5de4533ebddb5cc8, 6),
+    ('usdc-usd-coin', 'arbitrum', 'USDC.e', 0xff970a61a04b1ca14834a43f5de4533ebddb5cc8, 6),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:86]: arbiscan: https://arbiscan.io/token/0xff970a61a04b1ca14834a43f5de4533ebddb5cc8

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @kvtoraman

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**